### ISSUE TITLE
vm.$$ object is deleted in the 'leave' transition function

### ISSUE THREAD

**Issue by Samsy** [yyx990803/vue#697]
I'd like to access my dom object " overlay " in the vm.$$ element in the 'leave' transition function

template :
```<div class="application-component-overlay" v-el="overlay"></div>```

js : 

``` leave: function(el, done) {
         console.log(this.$$.overlay)
    }```

'this.$$' object is empty while entering in the leave function.

**Comment by agonbina** [comment]
You should replicate this behavior in a jsfiddle to make it easier for @yyx990803

**Comment by ayamflow** [comment]
Fiddle in duplicate #713.

**Comment by yyx990803** [comment]
Should be resolved with the same fix 67eb247
